### PR TITLE
add new stencil paper and update stencil section

### PR DESCRIPTION
**Pull request by bastianhagedorn**
I don't now if I did this correct or if there is a smarter way to update the website (made all the changes simply in vim) If there is any smarter way of updating websites let me know :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -93,6 +93,34 @@ <h2>Publications</h2>
...
 
+          <li>
+        <a href="publications/2018/hagedorn18Stencils.pdf">
+High Performance Stencil Code Generation with Lift
+        </a><br>
+        Bastian Hagedorn, Larisa Stoltzfus, Michel Steuwer, Sergei Gorlatch, Christophe Dubach
+        <a href="http://cgo.org/cgo2018/">
+International Symposium on Code Generationand Optimization (CGO) 2018
+        </a><br>		
+        <a href="publications/2017/hagedorn18Stencils.pdf" style="margin-right:1em">
```
Comment (naummo) [inline index.html]: Wrong year in the path here.

**Comment by naummo** [comment]
Cannot check if the link to the artifact is rendered in a good position, but the code looks right. Sorry for appropriating you into our university :)

File: index.html (hunk 1 above)
Comment (bastianhagedorn) [reply index.html]: the path will be introduced here https://github.com/lift-project/publications/pull/2

Comment (naummo) [reply index.html]: Yeah, but it is 2018/hagedorn18Stencils.pdf, not 2017/hagedorn18Stencils.pdf

**Comment by bastianhagedorn** [comment]
fixed the wrong year in the link to the stencil paper

**Pull request merged**